src/modules/blockchainApplication/connection/components/ConnectionSummary/index.js

```diff
@@ -51,7 +51,7 @@ const ConnectionSummary = () => {
   const rejectHandler = async () => {
     const result = await reject();
     addSearchParamsToUrl(history, {
-      modal: 'connectionSuccess',
+      modal: 'connectionStatus',
```

Comment:
Not related to this PR. 

I think the issue raised by the audit team concerning having pages that display information from query params as seen in this [PR](https://github.com/LiskHQ/lisk-desktop/pull/5174#issue-1802687812) would apply to this component. 
`<ConnectionStatus />` as seen from line `16 - 25`. I think we should take their suggestion by either limiting the `name` query param key or have a defined set of values. 

You can find the audit's suggestion in the audit final report `95. Phishing risk on the deviceDisconnectDialog modal`